server/src/main/java/io/littlehorse/server/streams/lhinternalscan/publicrequests/SearchExternalEventRequestModel.java

```diff
@@ -39,10 +42,10 @@ public class SearchExternalEventRequestModel
                 ExternalEventIdModel,
                 SearchExternalEventReply> {
 
-    public ExtEvtCriteriaCase type;
-    private WfRunIdModel wfRunId;
     private String externalEventDefName;
-    private Optional<Boolean> isClaimed = Optional.empty();
+    private Date earliestStart;
+    private Date latestStart;
+    private String externalEventDefId;
```

Comment:
We like to have the fields in the `Model` classes be the same type as the Proto. It would be best to have:

```
private ExternalEventDefIdModel externalEventDefId;
```

The proto should also be `ExternalEventDefId` and not `string`. Thanks!